packages/es-components/src/components/controls/radio-buttons/RadioGroup.js

```diff
@@ -8,10 +8,12 @@ import Fieldset from '../../containers/fieldset/Fieldset';
 import RadioButton from './RadioButton';
 
 const RadioFieldset = styled(Fieldset)`
+  margin-bottom: 20px;
   legend {
     border: none;
     font-size: 22px;
-    margin-bottom: 0;
+    line-height: 1;
```

Comment:
Is this line-height override necessary? Everything else looks fine... this just seems like such a minor spacing change over the default, and loses some breathing whitespace.